src/main/kotlin/no/nav/syfo/client/behandlendeenhet/BehandlendeEnhetClient.kt

```diff
@@ -61,7 +61,7 @@ class BehandlendeEnhetClient(
             valkeyStore.setObject(
                 key = cacheKey,
                 value = behandlendeEnhet,
-                expireSeconds = TWELVE_HOURS_IN_SECS
+                expireSeconds = ONE_HOUR_IN_SECS
```

Comment:
Dette er kallet som henter hvilken enhet en sykmeldt tilhører, for å sjekke om veileder har tilgang til den enheten?

Reply:
Ja, jeg ble litt usikker på dette. Men siden vi driver og endrer enhet nå så tenkte jeg at det er lurt å cache dette kortere.

Men vi må eventuelt komme tilbake til dette etter hvert - særlig hvis vi begynner å bruke `syfobehandlendeenhet` i flere tilfeller (nå er det vel egentlig bare Nav utland som hentes derfra, mens resten av kallene går rett til Norg).